aten/src/ATen/templates/LazyIr.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// This file contains autogenerated LazyTensor IR nodes
+${lazy_ir_sysinc}
+${lazy_ir_inc}
+
+namespace torch {
+namespace lazy {
+using at::operator<<;
+
+// kNullValue is used to contribute a static hash value any time
+// a node has an Optional<Value> input that is nullopt.  It is important
+// to differentiate between HASH(nullopt, something) and HASH(something, nullopt),
+// and using kNullValue in the hash function in the order of arguments
+// serves this purpose.
+static const torch::lazy::Value kNullValue = torch::lazy::Value();
```

Comment:
static value in header file? This is my suspicious face.

Reply:
oh yea, this looks really dumb.  I'll figure out why we did this and move it somewhere better..

Reply:
oh.. i remember now. This isn't too bad since all I was going for was to not call the Value() constructor a ton of times, but 2 different null values are ok to interchange.  Still, i'll fix it, it's ugly